src/forms/FormFieldAdvancedSelection/AdvancedSelection.styled.tsx

```diff
@@ -54,7 +54,8 @@ export const ChipsWrapper = styled.div`
 `;
 
 export const OptionsCheckedModalWrapper = styled.div`
-  margin-bottom: ${(pr) => (pr.isModalOpen ? "40px" : "")};
+  margin-bottom: ${({ isModalOpen }) => (isModalOpen ? "40px" : "")};
+	margin-top: ${({ isModalOpen }) => (isModalOpen ? "40px" : "")};

```

Comment:
This seems to have an extra tab / space, if this is the only issue I'll change it directly here.